### PR TITLE
[DELIVERABLE 2] SRS: Maintainability and Supportability Requirements

### PR DESCRIPTION
**Pull request by Ayushi1972**
## Description
Added maintainability requirements
Added supportability requirements
Added adaptability requirements

## How Has This Been Tested?
Compiled locally

### CODE REVIEW

File: docs/SRS/SRS.tex [hunk 1]
```diff
@@ -231,11 +231,51 @@ \subsection{Release Requirements}
...
+  \item \textbf{Requirement:} The tool must allow for easy rollback of updates in case of errors.
+     \begin{itemize}
+         \item \textbf{Rationale:} Quick rollback capabilities minimize downtime and user disruption in case an update introduces issues.
+         \item \textbf{Fit Criteria:} Any update can be reverted with minimal effort, ensuring the system returns to a stable state within one hour.
+     \end{itemize}
+     
+  \item \textbf{Requirement:} The tool must provide automated testing for all refactoring functions.
+     \begin{itemize}
+         \item \textbf{Rationale:} Automated testing ensures that changes do not introduce new bugs, maintaining the reliability and stability of the tool.
+         \item \textbf{Fit Criteria:} All refactoring methods have associated unit tests that run automatically with each code change, ensuring 95% code coverage.
```
Comment (Sevhena) [inline docs/SRS/SRS.tex:256]:
```suggestion
         \item \textbf{Fit Criteria:} All refactoring methods have associated unit tests that run automatically with each code change, ensuring 95\% code coverage.
```

Comment (Sevhena) [inline docs/SRS/SRS.tex:256]: I believe 95% code coverage is a bit much. The prof and other source recommend like 80%

**Review by Sevhena — CHANGES_REQUESTED** [review]

**Review by Sevhena — APPROVED** [review]
LGTM

**Review by mmyaaaaa — APPROVED** [review]
LGTM

**Review by Sevhena — APPROVED** [review]
LGTM

**Pull request merged**